usr/libexec/greenboot/greenboot-service-monitor

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -eo pipefail
+set -o pipefail
```

Comment:
why this? :thinking:

Reply:
`state="$(systemctl is-enabled $service)"` exits>0 if any service is disabled, so removed -e.

Reply:
if you put the subshell `$(..)` in the if statement it shoulnd' fail right?

Reply:
yes, that worked, updated the script.